src/site/pages/digital/src/utils/CircuitInfo/DigitalCircuitInfoHelpers.ts

```diff
@@ -30,7 +30,7 @@ export function GetDigitalCircuitInfoHelpers(store: AppStore, canvas: RefObject<
             const open = circuit.isSaved || window.confirm(OVERWRITE_CIRCUIT_MESSAGE);
             if (!open) return;
 
-            const circuitData = await getData();
+            const circuitData = await getData() as string;
```

Comment:
What is it otherwise?

Reply:
`unknown`

Reply:
This is implies something else is probably wrong